flex/03-flex-header-2/README.md

```diff
@@ -4,7 +4,7 @@ We're starting to sneak in a little more CSS that you haven't seen yet. Don't wo
 
 For this one you will probably need to edit the HTML a little bit. Often with flexbox you need to add containers around things to make them go where you need them to go. In this case, you probably want to separate the items that go on the left and right of the header.
 
-This is also the first example where you'll be nesting flex containers inside each other.
+Just like you did in the previous exercise with the `ul`, you'll be nesting flex containers inside each other.
```

Comment:
```suggestion
Just like you did in the `02-flex-header` exercise, you'll be nesting flex containers inside each other.
```
Just a slight tweak to the change, otherwise this looks good!

Reply:
Sorry about that stupid question, but how can I accept your change? Lol

Reply:
So you can either just make this change locally, commit and push like you normally would, or you should be able to click the "Commit suggestion" button just under the codeblock above:

![Commit suggestion example](https://user-images.githubusercontent.com/70952936/214069197-ce6db2d0-3ea0-4a35-b65a-41d682158fca.png)

(if you don't see that button, then copy+pasting the suggestion and making the change locally will be fine, though)

Reply:
Thx for the help, I saw your suggestion on the mobile app, probably this is the reason that I don't see this button, when I get home I will commit it!

Reply:
Ah yeah that could be it. No rush, though, and thanks for making the update!